docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,496 @@
+
+### Google OIDC and Team Sync support
+
+<!-- Jo Guerreiro, AuthNZ -->
+
+_Generally available in all editions of Grafana_
+
+Grafana now supports Google OIDC through the `Google` OAuth provider in addition to the existing `Google` OAuth2 provider. This allows you to use Google OIDC to authenticate users in Grafana, which in turn, lets Grafana reduce the access scope to only the required scopes for authentication and authorization.
+
+This release also adds support for Google OIDC in Team Sync. You can now easily add users to teams by using their Google groups.
+
+To learn how to migrate your Google OAuth2 setup to OIDC and how to set up Team Sync, refer to our [Google authentication documentation]({{< relref "../setup-grafana/configure-security/configure-authentication/google/" >}}).
+
+## Plugins
+
+### Angular deprecation changes
+
+<!-- Giuseppe Guerra, Plugins Platform -->
+
+_Generally available in all editions of Grafana_
+
+We've made the following updates to increase awareness of the [Angular deprecation]({{< relref "../developers/angular_deprecation/" >}}) and its consequences in future releases of Grafana:
+
+#### UI changes
+
+- Added an **Angular** badge next to affected plugins in the plugins catalog.
+- Added an alert at the top of a plugin's page in the plugins catalog when browsing Angular plugins.
+- Added an alert at the top of the query editor when editing panels that use Angular data source plugins.
+
+#### Other changes
+
+- Angular Plugins will not be loaded if [angular_support_enabled]({{< relref "../setup-grafana/configure-grafana/#angular_support_enabled" >}}) is set to `false`.
+
+Learn more in our [Angular support deprecation documentation]({{< relref "../developers/angular_deprecation/" >}}).
+
+### Deprecated provisioning of data sources with invalid UIDs
+
+<!-- Giuseppe Guerra, Plugins Platform -->
+
+_Generally available in all editions of Grafana_
+
+Grafana now logs an error when provisioning data sources with invalid UIDs. A valid UID is a combination of a-z, A-Z, 0-9 (alphanumeric), `-` (dash), and `_` (underscore), with a maximum length of 40 characters.
+
+Provisioning data sources with invalid UIDs will be removed in future versions of Grafana, and will return an error instead.
+
+## Subfolders: folder selection
+
+<!-- Zsofia Komaromi -->
+
+_Available in public preview in all editions of Grafana_
+
+When saving or moving a dashboard, you can now see the full folder tree when selecting the destination folder.
+
+To get started creating subfolders, enable the `nestedFolders` feature toggle. We recommend that you enable this feature only on test or development instances, rather than in production environments.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10.1-subfolders-folder-picker.png" max-width="750px" caption="Selecting a folder in Grafana" >}}
+
+## Activate draft reports
+
```

Comment:
@rhorvath - Should this be under Dashboards and visualizations?

**Note, please don't commit any changes without checking in with Quynton Johnson first.**

Reply:
Good point, yes, we can move this under Dashboards section